server/routes/api.py

```diff
@@ -1,9 +1,33 @@
 from flask import Blueprint, jsonify, make_response
-
+from firebase_admin import auth, credentials
+import firebase_admin
+from flask import request
+from .create_credential_json import get_fbs_prv_key
 
 bp = Blueprint('api', __name__, url_prefix='/service')
 
+# creates the json
+fbs_private_key = get_fbs_prv_key()
+
+cred = credentials.Certificate(fbs_private_key)
```

Comment:
I got a ValueError here when executing `flask run`

![image](https://user-images.githubusercontent.com/50651099/153783387-0f70239f-2765-4e74-8d14-3aaea364a43f.png)

Reply:
Have you used the updated environment file? This error indicates that it couldn't read any environment variable. It's not related to it

Reply:
yep it works now!